### PR TITLE
Infocenter updates

### PR DESCRIPTION
**Pull request by florianesser**

### CODE REVIEW

File: doc/plugins/eu.esdihumboldt.hale.doc.user/toc.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<toc label="hale studio User Guide">
+<toc label="hale»studio User Guide">
```
Comment (stempler) [inline doc/plugins/eu.esdihumboldt.hale.doc.user/toc.xml:2]: Wrong or just not correctly displayed here?

**Review by stempler — APPROVED** [review]
LGTM, one comment on potential encoding issue.

File: doc/plugins/eu.esdihumboldt.hale.doc.user/toc.xml (hunk 1 above)
Comment (stempler) [reply doc/plugins/eu.esdihumboldt.hale.doc.user/toc.xml:2]: Interesting - looks different in "Conversation" than in "Files changed"

Comment (florianesser) [reply doc/plugins/eu.esdihumboldt.hale.doc.user/toc.xml:2]: I checked the encoding and it looks good to me in UTF-8. Not sure why it displays wrong on the "Files changed" tab.

**Pull request merged**